src/graphics/programlib/programlib_phong.js

```diff
@@ -277,7 +277,7 @@ pc.programlib.phong = {
             } else {
                 code += chunks.specularAaNonePS;
             }
-            code += this._addMap("specular", options, chunks, uvOffset);
+            code += this._addMap(options.metalnessWorkflow? "metalness" : "specular", options, chunks, uvOffset);

```

Comment:
What I think is not intuitive is metalness overriding specular. Metalness is not necessarily better, but it's different. What happens if you set both metalness and specular? That won't make sense. We need a switch that'll show either specular or metalness controls, useMetalness is that switch.

I agree on consistency, I'll change it.